src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+
+    },
+    removeBook: () => {
+
+    },
+  },
+});
+export default booksSlice.reducer;
```

Comment:
- [ ] Kindly make sure to export the actions from your slice because you will need them in your components later to mutate the app state:

`export const { addBook, removeBook } = booksSlice.actions`